### ISSUE TITLE
BUG: Client MUST restart their game in order to reconnect to a dedicated server.

### ISSUE THREAD

**Issue by Jetpack-Cat** [txnimc/KryptonReforged#7]
Clients can connect to a dedicated server, but if they disconnect, they have to restart their game in order to reconnect to the server.

Removing Krypton Reforged fixes the issue.

Krypton reforged is on both the client and server.
I'm running the server on the same pc I'm playing on.

Forge 36.2.35
MC: 1.16.5

**Comment by f0kx** [comment]
yeah same here after i disconnect and came back i need 10-20 trys to connect to the server again...restarting the whole game fixed it

**Comment by magsum** [comment]
This issue has been persistent for me for almost 2 months, for me.

**Comment by ITCMD** [comment]
Same issue here.

**Comment by Jetpack-Cat** [comment]
This mod doesn't do as much as it says, it's a junky port that gives no clear improvement. I deleted the mod and everyone playing my modpack was happier for it.

**Comment by MrSataana** [comment]
Holy... I just spend a whole week, 8 hours daily trying to figure out why I could only connect once per client session.. (one of my players doesn't have this issue, so id be interested to find out why that is... hardware, IP? Same ISP so 🤷‍♂️ )
Flushed DNS, reset models, did port forwarding, tried hosting a server on another computer in the same network.
Installing other "performance / tweak / fix" mods, uninstalling all content mods one by one and starting a server up trying to connect until eventually I got to this.

Please, for the love of god, ether remove the mod from CF or maintain / fix it, please <3

**Comment by Jetpack-Cat** [comment]
The guy's making money off this shitty mod he won't remove it lol

**Comment by Kasualix** [comment]
Any log pls?

**Comment by Xiantrius** [comment]
same problem client restart game before again connected in server.
Please fix this in all Minecraft versions.

**Comment by AKNALOGIAa** [comment]
2024  still this error
fuck this shit
